debezium-server-name-mapper/pom.xml

```diff
@@ -0,0 +1,69 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>io.debezium.examples</groupId>
+    <artifactId>debezium-server-name-mapper</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+
+        <database.hostname>localhost</database.hostname>
+        <database.port>3306</database.port>
+        <database.user>debezium</database.user>
+        <database.password>dbz</database.password>
+        <database.whitelist>inventory</database.whitelist>
+        <table.whitelist>inventory.customers</table.whitelist>
+
+        <kinesis.region>eu-central-1</kinesis.region>
+
+        <mysql.init.timeout>30000</mysql.init.timeout>
+
+        <version.debezium>1.2.0-SNAPSHOT</version.debezium>
+        <version.debezium.tag>1.2</version.debezium.tag>
+        <version.quarkus>1.5.0.Final</version.quarkus>
+    </properties>
+
+    <dependencyManagement>
+        <dependencies>
```

Comment:
Instead of using Debezium Server as a dependency, could we rather build this as a simple JAR (i.e. no Quarkus app itself) and then show how the built JAR is added to the lib folder of the Quarkus application? Would this not be a more common usage?

Reply:
@gunnarmorling IMHO we can't. Is not it that way that DI in Quarkus is resolved in build time not in runtime?

Reply:
Yes, you're right. That was a bit of a brain fart on my end :)